### PR TITLE
Add resnet

### PR DESCRIPTION
**Pull request by Simonexc**

### CODE REVIEW

**Review by Bartosz7 — COMMENTED** [review]
change MNIST to CINIC

**Review by Bartosz7 — CHANGES_REQUESTED** [review]
change mnist to cinic

File: src/model.py [hunk 1]
```diff
@@ -151,6 +196,8 @@ def training_step(self, batch, batch_idx):
         xs, ys = batch
         logits, loss = self.loss(xs, ys)
         preds = torch.argmax(logits, 1)
+        if self.config.cutmix_add != "none":
+            ys = torch.argmax(ys, 1)
```
Comment (Bartosz7) [inline src/model.py:194]: not argmax, need to change to multi-label accuracy
